client/contracts/truffle/contracts/kns/KNSRegistryV1.sol

```diff
@@ -0,0 +1,108 @@
+pragma solidity 0.4.24;
+
+import "./KNS.sol";
+import "../proxy/Initializable.sol";
+
+/**
+ * The KNS registry contract.
+ */
+contract KNSRegistryV1 is KNS, Initializable {
+    struct Record {
+        address owner;
+        address resolver;
+        uint64 ttl;
+    }
+
+    mapping (bytes32 => Record) records;
+
+    // Permits modifications only by the owner of the specified node.
+    modifier only_owner(bytes32 node) {
+        require(records[node].owner == msg.sender);
+        _;
+    }
+
+    /**
+     * @dev Constructs a new KNS registrar.
+     */
+    constructor() public {
```

Comment:
Same here, is it needed the normal constructor?

Reply:
Now I see that some of this contracts are needed for the only purpose of testing, can it be?